go/test/endtoend/tabletmanager/tablet_health_test.go

```diff
@@ -131,7 +126,7 @@ func TestHealthCheck(t *testing.T) {
 
 	// stop replication, make sure we don't go unhealthy.
 	// TODO: replace with StopReplication once StopSlave has been removed
-	err = clusterInstance.VtctlclientProcess.ExecuteCommand("StopSlave", rTablet.Alias)
```

Comment:
This was intentionally left in so that we have test coverage for the old command.